### ISSUE TITLE
Biz Dev & Partnerships for QF

### ISSUE THREAD

**Issue by surbhiaroraa** [Giveth/giveth-planning#1659]
Objective: Get Money for QF

For context putting a screenshot of the discussion and proposal we had during GIVconnect. 
<img width="615" alt="Screenshot 2023-11-17 at 7 57 49 PM" src="https://github.com/Giveth/giveth-planning/assets/140404865/ae1c10d2-55e0-4bd0-8127-dbd5f8b0150c">
cc: @oyealmond @missgene 

@laurenluz @GriffGreen , since you have been leading the effort of getting the QF matching pool funds, can you help me with all the information you have, the pitch you used, and any deck or document you used while pitching/proposing to Optimism and Polygon for QF matching pool. It will just help to get a headstart on this.

**Comment by GriffGreen** [comment]
Some of this data is in here: 


# TL;DR of this alternative proposal

Giveth proposes to launch three Aragon-specific Quadratic Funding Rounds focused on supporting, as well as attracting builders and innovators within the Aragon ecosystem. Beyond just distributing funds, Quadratic Funding has proven to produce viral marketing campaigns, and this proposal hopes to bring this impact to the Aragon Ecosystem. To pull off this effort, incentivizing decentralized Aragon development and marketing the Aragon Ecosystem, Giveth seeks 295,000 USDC to fund and operate these 3 Aragon specific QF rounds. \~**80% of the funds will be given directly to Aragon builders**, the other 20% will be used run marketing campaigns, operate the rounds, and complete the required sybil analysis. 

# I. Executive Summary

**Project Name:**
Quadratic Funding Rounds for Builders

**Introduction:**
Giveth has been a friend of the Aragon Ecosystem since the very beginning. We collaborated with Aragon to create the ANT token and run the Aragon ICO. We also for 6 months ran an Aragon Dev team called the Aragon DAC and we have been using several Aragon DAOs on several chains still to this day.

Giveth’s primary product is an open-source web3 donation platform that operates on Ethereum Mainnet, Gnosis Chain, Polygon, Optimism, Ethereum Classic, and Celo, We provide a seamless P2P crowdfunding experience, free from fees and intermediaries. We are a hub for nearly three thousand individuals and initiatives, all geared towards building public goods and we have recently added Quadratic Funding (QF) to the platform.

QF is not your average grant program. While most grant programs can only give the funding that they have, QF incentivizes the community to pitch in and help to fund the development they want to see. It is not uncommon to see rounds where there is about as much donated as there are funds in the matching pool. Not only does QF attract more funding, but it also unleashes a viral marketing campaign, where the projects themselves are incentivized to get loud about their work and promote themselves and, in this case, Aragon, to anyone that will give them $1.

We are proposing to run 3 Aragon-specific Quadratic Funding Rounds in the coming year of 2024. These rounds, starting with a $25,000 matching pool, will escalate in size, culminating with a $150,000 round. This is designed to amplify engagement and attract a new wave of developers to Aragon's DAO ecosystem. The network effect that this will bring to Aragon’s open source development can’t be understated. Each round will be intentionally curated by the Giveth team with support from Aragon core team members. This effort is not intended to fund DAOs that are USING Aragon, but instead fund developers that are BUILDING with the Aragon tech stack.

\
**Funding Request:** 

295,000 USDC 

\
**Payout Address:** 

giv.eth (0x4d9339dd97db55e3b9bcbe65de39ff9c04d1c2cd) 

\
**Duration:**
The rounds will span throughout the entirety of the year 2024.

**Key Objectives:**
Outline the main goals and objectives of your project.

* **Fostering Ecosystem Growth**: To steadfastly support projects within the Aragon ecosystem, emphasizing those innovating DAOs, dApps, and plugins on Aragon's progressive tech stack.
* **Strategic Impact Cultivation**: To attract and support projects that embody Aragon's mission and values, enhancing the digital landscape with transformative change.
* **Community Empowerment**: To fortify the broader Aragon community, laying the groundwork for a future of decentralized development.
* **Marketing and Awareness of Aragon**: QF rounds are not just funding sources, they are viral marketing campaigns. Not only will Giveth market the round to the DAO builder community, but every single project in the round is incentivized to create a marketing campaign about their own project, and all these efforts will effectively market Aragon as a leader in DAO development software.

**Timeline:**

**Q4 2023 - Preparation of Round 1**

Preparation Phase: Set eligibility criteria, onboard new and existing projects, initialize the marketing campaign, and curate applicants for Aragon's inaugural QF round.

**Q1 2024 - Round 1 Execution and Preparation of Round 2**

Marketing Phase: Continue Marketing for Round 1.
Execution Phase: Ensure the seamless execution of QF Round 1, projects will collect donations from the community, and at the end of the round we will conduct the Sybil analysis.

Fund Distribution: Distribute 25,000 USDC from the Aragon Matching Pool to Round 1 recipients.

Feedback Phase: Collect feedback from all stakeholders in Round 1 to improve for Round 2.

Preparation Phase: Onboarding and marketing to attract new old projects and open applications for Round 2.

**Q2 2024 - Continue Marketing for Round 2, Round 2 Execution and Massive Marketing Campaign for Round 3**

Marketing Phase: Continue Marketing for Round 2.
Execution Phase: Ensure the effective execution of QF Round 2, projects will collect donations from the community, and at the end of the round we will conduct the Sybil analysis.

Fund Distribution: Distribute 60,000 USDC from the Aragon Matching Pool to Round 2 recipients.

Feedback Phase: Collect feedback from all stakeholders in Round 2 to improve for Round 3.

Marketing Phase: Initialize Massive Marketing for Round 3.

**Q3 2024 - Massive Marketing Campaign over the Summer**

Marketing Phase: Continue Massive Marketing Campaign for Round 3.
Preparation Phase: Onboarding campaign to attract new & existing projects and open applications for Round 3.
Note: Things naturally slow down over the summer so we don’t believe that it’s strategic to run a development-focused round in Q3.

**Q4 2024 - Round 3 Execution and Presentation of a Report**

Marketing Phase: Continue Massive Marketing Campaign for Round 3.

Execution Phase: Ensure the success of QF Round 3, projects will collect donations from the community, and at the end of the round we will conduct the Sybil analysis.

Fund Distribution: Distribute funds from the Aragon Matching Pool to Round 3 recipients.

Feedback Phase: Compile a comprehensive report for Aragon, summarizing the impact of all rounds and key findings.

## II. Budget Details

**Funding Allocation:**

Round 1: 25,000 USDC

Round 2: 60,000 USDC

Round 3: 150,000 USDC

Sybil detection, marketing & admin costs: 60,000 USDC

Total Budget: 295,000 USDC

## III. Expected Outcomes and Impact

Measurable Metrics will include:

* The number of projects funded.
* Total funding collected beyond the Matching Pools.
* User engagement metrics and community feedback.
* Sybil analysis results.
* Success stories for new products within the Aragon ecosystem that were funded by QF.

## IV. Team and Expertise

Giveth's team is a dedicated group of individuals with the skills and expertise necessary to ensure the success of the Quadratic Funding Rounds and effectively onboard the Aragon community. Key team members include:

* Lauren: Giveth's product lead and communications advisor, known for spearheading major Giveth initiatives and campaigns. She's a visionary leader in the Giveth DAO and a problem-solving advocate.
* Mitch: Serving as DAO Ops Lead and Product Manager at [Giveth.io](http://Giveth.io), Mitch brings extensive experience from impactful projects like Commons Stack, TEC, and more.
* Griff Green: A co-founder of Commons Stack, Giveth, General Magic, and DAppNode, Griff is a prominent figure in the Ethereum community and a top steward in various Ethereum projects.
* Cherik: Giveth's Lead Front-End Developer, instrumental in enhancing the front-end design of Giveth products.
* Ashley: A versatile team member handling multiple roles, including stewarding the Communitas working group and onboarding processes.
* Almond: Focused on community building and communications, Almond strengthens Giveth's connections and outreach.
* Brichis: Fundraiser at Giveth, passionate about funding mechanisms for public goods and governance.
* Gitcoin Passport: We have integrated Gitcoin passport as our first line of sybil defense
* Trusta Labs: Our partner for sybil detection, reviews every round to find sybil attacks

## V. Results from our previous Quadratic Funding Rounds

**Quick Summary of Outcomes from Our Alpha Round (July 18- August 1st, 2023)**

* Matching Pool: 10,000 xDAI on Gnosis Chain
* Community Contributions: $11,737 in donations from the community.
* Total funds raised for projects: $21,737, More than double the Matching Pool!
* Project Engagement: 47 verified projects benefited from the initiative.
* Donor Participation: 246 unique donors contributed.
* Donation Activity: 696 eligible donations were made, with a total of 919 donations received, indicating a high level of engagement.
* Integrity Measures: 103 ineligible donations were removed, maintaining the integrity of the funding process, thanks to Gitcoin Passport.
* Sybil Defense: 35 sybil addresses were identified and removed with the assistance of Trustalabs, ensuring a fair and transparent allocation of funds.

**Quick Summary of Outcomes from our Optimism Round (October 9-23, 2023)**

* Matching pool: 25,000 DAI on Optimism Network
* Donations raised from individual donors: $22,996
* Total funds raised for projects: $47,996, almost double the Matching Pool!
* Verified projects: 125
* Total number of donations: 1728
* We are still doing the sybil analysis, so the rest of the numbers will be updated on this forum post one the analysis is done this week.

## Useful links

Website: <https://giveth.io/>

Twitter: <https://twitter.com/giveth>

Instagram: <https://www.instagram.com/giveth.io/>

Youtube: <https://www.youtube.com/givethio>

Discord: <https://discord.giveth.io/>

Github: <https://github.com/Giveth/>

**Contact Information:**
For further inquiries, please reach out directly to Brichis as I might be difficult to reach via these next few weeks while I attend the conferences in Istanbul.
Telegram/Discord: @brichis
Twitter: @brichis_
E-mail: [brichis@generalmagic.io](mailto:brichis@generalmagic.io)

**Comment by laurenluz** [comment]
@surbhiaroraa you can find all the info you need on the forum in our summary posts.

Alpha Round: https://forum.giveth.io/t/qf-alpha-round-results/1149
Optimism: https://forum.giveth.io/t/qf-optimism-round-results-october-9-23-2023/1226
